File: leetcode2/0412. Fizz Buzz.py
class Solution:
    def fizzBuzz(self, n):
        """
        :type n: int
        :rtype: List[str]
        """
        rep=[]
        for i in range(1,n+1):
            if i%3!=0 and i%5!=0:
                rep.append(str(i))
            elif i%3==0 and i%5!=0:
                rep.append('Fizz')
            elif i%3!=0 and i%5==0:
                rep.append('Buzz')
            else:
                rep.append('FizzBuzz')
        return rep

File: archives/C#/0412. Fizz Buzz.cs
public class Solution {
    public IList<string> FizzBuzz(int n) {
        IList<string> rep=new List<string>();
        for(int i=1;i<=n;i++){
            if(i%3==0 && i%5==0){
                rep.Add("FizzBuzz");
            }
            else if(i%3==0){
                rep.Add("Fizz");
            }
            else if(i%5==0){
                rep.Add("Buzz");
            }
            else{
                string tmp=Convert.ToString(i);
                rep.Add(tmp);
            }
        }
        return rep;
    }
}
